monitoring/notification/dashboard.py

```diff
@@ -0,0 +1,567 @@
+from grafanalib.core import (
+    ConstantInput,
+    DataSourceInput,
+    RowPanel,
+    Threshold,
+    StateTimeline,
+    Templating,
+    Template,
+    Heatmap,
+    HeatmapColor,
+    YAxis,
+)
+from grafanalib import formatunits as UNITS
+from scalgrafanalib import(
+    layout,Target,
+    TimeSeries,
+    Dashboard,
+    Stat,
+    GaugePanel,
+    metrics,
+    Tooltip,
+)
+
+LABEL_EXPR = {
+    'notification-processors': '.*-(.*)-headless',
+    'topics': '.*backbeat-notification-(.*).*'
+}
+
+DEFAULT_JOB_PREFIX = 'artesca-data-'
+
+class Metrics:
+    DELIVERY_LAG = metrics.BucketMetric(
+        'notification_queue_processor_notification_delivery_delay_sec',
+        'target', 'job', 'status', namespace='${namespace}',
+    ).with_defaults(
+        'status="success"',
```

Comment:
since this is an equality, it can be written more simply inline with the label defintion (line 34), like what you did for namespace